pyrail/models.py

```
@@ -19,20 +20,114 @@ class Station(DataClassORJSONMixin):
+    duration: str  # Duration of the connection
+    vehicles: List[str] = field(default_factory=list)  # List of vehicle identifiers
+    departure_delay: int | None = field(
+        metadata=field_options(alias="departureDelay"), default=None
+    )  # Delay at departure in seconds
+    arrival_delay: int | None = field(
+        metadata=field_options(alias="arrivalDelay"), default=None
+    )  # Delay at arrival in seconds
+
+
+@dataclass
+class ConnectionsResponse(ApiResponse):
```

Ruff (0.8.2)

Missing docstring in public class

(D101)